spec/policies/user_policy_spec.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe UserPolicy do
+  let(:user) { create :user }
+  let(:admin) { create :user, :admin }
+  let(:other_user) { create :user }
+
```

Comment:
Please add a policy + spec for `email` to make sure only the user and admins can view his/her email address.